cocotb/decorators.py

```diff
@@ -334,7 +334,7 @@ def _wrapped_hook(*args, **kwargs):
             try:
                 return RunningCoroutine(self._func(*args, **kwargs), self)
             except Exception as e:
-                raise raise_error(self, str(e))
+                raise raise_error(self, "Hook raised error")
```

Comment:
To be consistent with the other cases above you might want to make this `raise raise_error(self, "Hook raised error:")` (or change the other two messages, of course)

Reply:
Done in #716